src/boards.c

```diff
@@ -380,6 +383,10 @@ void neopixel_write (uint8_t *pixels)
   nrf_pwm_seq_cnt_set(pwm, 0, sizeof(pixels_pattern)/2);
   nrf_pwm_event_clear(pwm, NRF_PWM_EVENT_SEQEND0);
   nrf_pwm_task_trigger(pwm, NRF_PWM_TASK_SEQSTART0);
+
+  // no need to blocking wait for sequence complete
```

Comment:
don't worry, I am part of the original writer. Wait for not, it should not affect the neopixel since we don't update it fast enough :)

Reply:
I wouldn't rely on the timing of calls to neopixel_write happening at a slow enough frequency to ensure that multiple calls aren't accidentally chained.

Although you know that is the case now, future changes may incorrectly assume the opposite. Instead, I'd check the timing at the start of the call to ensure the pause was long enough. That way callers of neopixel_write don't need to worry about it.

Reply:
reason I don't like to have blocking wait in bootloader since it could be troublesome. But this is hw, there should be none. I guess it could be safe anyway